<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>江苏海创系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_Main">
    
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox_Targets">
      <property name="title">
       <string>侦测目标 (TARGETS)</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tblTargets">
         <property name="columnCount">
          <number>4</number>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>机型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>距离</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>威胁度</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_Radar">
      <property name="title">
       <string>雷达</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widgetRadar" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #000000;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <item row="0" column="2" rowspan="2">
     <widget class="QGroupBox" name="groupBox_Control">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>作战控制</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_SystemStatus">
         <property name="text">
          <string>系统状态: 待机</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAutoMode">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>自动接管模式</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnJammer">
         <property name="text">
          <string> 手动干扰 </string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSpoof">
         <property name="text">
          <string> 手动诱骗 (SPOOF)</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_Log">
      <property name="title">
       <string> 系统日志 </string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextBrowser" name="textLog"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>